### PR TITLE
gentoo installation instructions

### PR DESCRIPTION
**Pull request by mbx162**
instructions recalled from my experience setting up a work-environment on a mbp16,2 inspired by the arch/installation.md

### CODE REVIEW

File: docs/distributions/gentoo/installation.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Installing Gentoo on a Mac with the T2 Chip
+
+You will need:
+
+- USB drive with at least 1GB
+- A way to plug it into your Mac (USB-C isn't USB-A)
+- A wired internet connection (i.e. USB-C to Enternet dongle) or wifi (not all models support it currently), check [the guide](https://wiki.t2linux.org/guides/wifi/) for compatibility and instructions. Its also technically possible to perform an offline installation, see [this](https://wiki.archlinux.org/index.php/Pacman/Tips_and_tricks#Installing_packages_from_a_CD/DVD_or_USB_stick) (retrieve the packages from an Arch virtual machine or Docker container)
```
Comment (networkException) [inline docs/distributions/gentoo/installation.md:7]: The arch instructions do not really apply here as you are using Ubuntu for the install process

File: docs/distributions/gentoo/installation.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- A usb keyboard (temporary)
+- About a weekend and some patience.
+
+---
+
+1. Create bootable media
+
+ * get the ubuntu live image to create a system used to chroot [https://github.com/marcosfad/mbp-ubuntu](https://github.com/marcosfad/mbp-ubuntu)
```
Comment (networkException) [inline docs/distributions/gentoo/installation.md:15]: Try to use capital letters for the beginning of list items as well as names such as "Ubuntu"

File: docs/distributions/gentoo/installation.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+ * dd on usb stick
+
+2. Boot into osx
```
Comment (networkException) [inline docs/distributions/gentoo/installation.md:18]: Please use the name "macOS" instead of osx

File: docs/distributions/gentoo/installation.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+4. boot into usb gentoo
+
+ * make sure apple-bce is loaded on boot (do-load apple-bce)
+ * verify everything is working
+ * create stage4
+
+5. boot into ubuntu
+
+ * install ubuntu (just for the boot partition) or you modify
+   /dev/nvme0n1p0 with the appropriate files
```
Comment (networkException) [inline docs/distributions/gentoo/installation.md:42]: Its a good practise to use fixed width sections (code blocks) for file system paths

**Review by networkException — CHANGES_REQUESTED** [review]
Please fix the issues I and the linter pointed out

You also need to add any new article to the [mkdocs config](https://github.com/t2linux/wiki/blob/master/mkdocs.yml) so that it gets picket up by the navigation and search system

File: docs/distributions/gentoo/installation.md (hunk 4 above)
Comment (Redecorating) [inline docs/distributions/gentoo/installation.md:42]: I think it numbers partitions starting from 1 (`/dev/nvme0n1p1`), unless gentoo does it differently, so nvme0n1p0 may not exist.

File: docs/distributions/gentoo/installation.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+ * install stage4
+
+7. boot into macbook gentoo
+
+ * cleanup /boot
+ * continue setting up your system
+
+8. make gentoo default os
+
+ * press shift in the osx-boot select (small reload icon is displayed under the partition image)
```
Comment (Redecorating) [inline docs/distributions/gentoo/installation.md:57]: Is this shift or option (or both)? The startup manager guide has the option key for this.

**Review by Redecorating — CHANGES_REQUESTED** [review]

File: docs/distributions/gentoo/installation.md (hunk 5 above)
Comment (AdityaGarg8) [reply docs/distributions/gentoo/installation.md:57]: I guess you mean control

**Comment by Jcbjoe2021** [comment]
> instructions recalled from my experience setting up a work-environment on a mbp16,2 inspired by the arch/installation.md

What's the status of gentoo on the t2 mbp? I would really like to install gentoo

**Comment by EmreTech** [comment]
#211 (merged) replaces this.

**Comment by AdityaGarg8** [comment]
Can this be closed now?

**Pull request closed without merging**